### PR TITLE
Construction of the mobile version of the portfolio.

### PR DESCRIPTION
**Pull request by sinruidotec**
**I am working on improving the portfolio project and its features.**

**In this Milestone I:**

- structured de HTML and CSS files.
- added experience and about sections.
- Added Assets Files with SVG vectors downloaded directly from Figma to this project.
- Add h1, h2, h3, ul semantic tags.
- Import necessary fonts to build this project.
- Linters added.

**_Waiting for new changes that are required._**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,179 @@
...
+                        dummy
+                        text.</p>
+                    <ul class="multi-post__stack">
+                        <li class="stack__css">CSS</li>
+                        <li class="stack__html">HTML</li>
+                        <li class="stack__bootstrap">Bootstrap</li>
+                        <li class="stack__ruby">Ruby</li>
+                    </ul>
+                    <button type="button" class="experience__button">See project</button>
+                </div>
```
Comment (DuaneDave) [inline index.html:64]: According to the project requirements, the work cards should be aligned using Flexbox in order to have more options for alignment. I advise you to use this in order to stick with the requirements

**Review by DuaneDave — CHANGES_REQUESTED** [review]
Hi @sinruidotec 👋 , 

Good job so far on this project!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✔️ Good job, your styling looks great
✔️ Your HTML is very semantic

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
N/A

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (sinruidotec) [reply index.html:64]: I have created the experience__multi-post class to use the flexbox property in the section indicated by the code reviewer and thus comply with the requirement of the project in which you have to create a card with flexbox to organize its elements and use a grid to accommodate the rest of the cards.

**Review by Efiamotu-1 — APPROVED** [review]
Hi @sinruidotec, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

# STATUS: APPROVED 🕺🥳 

giphy


## Highlights :dart: 
* Correct GitHub flow :heavy_check_mark: 
* Good commit messages :heavy_check_mark: 
* All linters passing :heavy_check_mark: 
* Good PR Title & summary :heavy_check_mark: 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

## Important Notes 📝

- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

> **Note:** Please, you can now Merge since it is Approved ✅, However if there are any conflicts please resolve them before merging.


## Thank you for your time 🙏 and call me if you need help or clarification ☎️

- This is My **GitHub**: [@Efiamotu-1](https://github.com/Efiamotu-1/)
- This is My **linkedIn**.[Habeeb Efiamotu Musa](https://www.linkedin.com/in/musa-habeeb/)
- This is My **Twitter**: [@EFYAMOTU](https://twitter.com/EFYAMOTU)
- This is My **Slack**: [Habeeb Efiamotu Musa](https://slack.com)

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**